packages/spark-extras/components/dictionary/_dictionary.scss

```diff
@@ -5,17 +5,18 @@
 //
 // Available Variables
 ///
-$dictionary-border: 1px solid $gray-medium !default;
+$dictionary-border: 1px solid $gray-light !default;
 $dictionary-stripe-color: $gray-light !default;
 $dictionary-breakpoint: 38.4375rem !default;
 
-.sprk-c-Dictionary--striped {
+.sprk-c-Dictionary {
   border: $dictionary-border;
+  padding: $space-m;
 }
 
 .sprk-c-Dictionary__keyvaluepair {
   width: 100%;
-  padding: $space-inset-m;
+  padding: $space-m / 2;
```

Comment:
Why the division by 2? In this case, 16 / 2 is 8, which is $space-s.

Reply:
in the case of the striped dictionary, i need to have space-m space between, but divided between the top and bottom. I could use space-s instead, but if space-s changes and space-m doesnt it would break. (Not sure thats a real thing that would happen)